.github/workflows/stales.yml

```diff
@@ -20,3 +24,5 @@ jobs:
         days-before-stale: 30
         days-before-close: 5
         remove-stale-when-updated: true
```

Comment:
Based on https://github.com/actions/stale, `remove-stale-when-updated` should cover both `remove-issue-stale-when-updated` and `remove-pr-stale-when-updated`?

Reply:
I was thinking the same at first (since it mentions override), but I believe that the two ones added here also triggers on comments, when the former only triggers if someone push a new commit to the PR. If I recall, we've always have to manually remove the stale label when someone has added a comment to the tickets. Based on my comment here and if you re-read again, do you interpret it the same way as I do?

Reply:
On the other hand, it might simply be that this hasn't worked because we've had older versions of the stale plugin and not having the "permissions" enabled. So, maybe these lines are superflous in the end.

Reply:
> Based on my comment here and if you re-read again, do you interpret it the same way as I do?

Yes, I think you're right. `remove-stale-when-updated` only mentioned 'updates', not 'updates/comments'. Sorry for the noise.